projects/sd-jwt-dotnet.md

```diff
@@ -0,0 +1,68 @@
+# Project Name
+
+SD-JWT-DotNet
+
+# Preferred Maturity Level
+
+Labs
+
+# Project Description
+
+A Dot Net implementation of the [Selective Disclosure for JWTs](https://www.ietf.org/archive/id/draft-ietf-oauth-selective-disclosure-jwt-04.html) (SD-JWT) specification.
+
+# Alignment with the OpenWallet Foundation Mission
+
+This library implements the Selective Disclosure for JWTs specification, which is an important building block for future digital identity wallets.
+
+# Code of Conduct
+
+None
+
+# TAC Sponsor
+
+None
+
+# Project License
+
+Apache 2.0.
+
+# Source Control
+
+[https://github.com/thomas-tran/sd-jwt-dotnet](https://github.com/thomas-tran/sd-jwt-dotnet)
+
+# Issue Tracker
+
+[https://github.com/thomas-tran/sd-jwt-dotnet/issues](https://github.com/thomas-tran/sd-jwt-dotnet/issues)
+
+# External Dependencies
+
+```nuget
+// https://www.nuget.org/packages/gfoidl.Base64
+// License: MIT
+gfoidl.Base64
+
+```
+
+# Release Methodology
+
+N/A
+
+# Initial Maintainers
+
+Thomas Tran([Github](https://github.com/thomas-tran))
+
+# Proposed Project Governance
+
+None
+
+# Links to Documented Governance Practices
+
+https://docs.google.com/forms/u/2/d/e/1FAIpQLSeO1bDGHUP-ZpCo1uynm94YOxZlek6RhCH7o3FnX1lZSXXfSQ/formResponse?pli=1
```

Comment:
Is the form submitted? Do we have access to the submission?

Reply:
I've completed the form submission, but unfortunately, I haven't received any response from the Linux Foundation. I'm unsure about the next steps and would appreciate guidance on what to do next.

Reply:
Hi @thomas-tran, the next step is to join the next TAC meeting to discuss this proposal and the comments we collected in this PR. The next call is scheduled for Nov 29th, 7:00am Pacific Time (GMT-08:00)

* https://tac.openwallet.foundation/meetings/
* https://discord.com/channels/1022962884864643214/1080192375621685288